development/nautobot_config.py

```diff
@@ -363,3 +363,6 @@ def is_truthy(arg):
     EXTRA_INSTALLED_APPS.append("debug_toolbar")
 if "debug_toolbar.middleware.DebugToolbarMiddleware" not in settings.MIDDLEWARE:
     settings.MIDDLEWARE.insert(0, "debug_toolbar.middleware.DebugToolbarMiddleware")
+
+# Increasing limit of Celery run time, because some dump examples are intense!
+NAUTOBOT_CELERY_TASK_SOFT_TIME_LIMIT = 600
```

Comment:
1) `NAUTOBOT_CELERY_TASK_SOFT_TIME_LIMIT` is an environment variable - the settings variable you need here would be simply `CELERY_TASK_SOFT_TIME_LIMIT`
2) Also need to set the hard limit `CELERY_TASK_TIME_LIMIT` correspondingly.
3) We should document the need for this in the README so that any end users of this plugin don't get tripped up by it.

Reply:
sorry, should not be committed